main.py

```diff
@@ -169,16 +169,26 @@ def send_email(request: Request) -> Tuple[str, int]:
         response.raise_for_status()
         log_info("notify email requested", **log_context)
     except RequestException as error:
-        notify_error = error.response.json()["errors"][0]
-        status_code = error.response.status_code
-        message = "notify request failed"
-        log_error(
-            message,
-            **log_context,
-            notify_error=notify_error,
-            status_code=status_code,
-        )
-        return message, error.response.status_code
+        try:
+            notify_error = error.response.json()["errors"][0]
+            status_code = error.response.status_code
+            message = "notify request failed"
+            log_error(
+                message,
+                **log_context,
+                notify_error=notify_error,
+                status_code=status_code,
+            )
+            return message, error.response.status_code
+        except AttributeError:
```

Comment:
The exception here should likely be immediately after the code that could trigger it - e.g. `notify_error = error.response.json()["errors"][0]`

Reply:
As `requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))` was the error that was seen in testing, I think we should be explicitly catching the `ConnectionError` before `RequestException`. This would be clearer and cover off any other connection errors.